src/elements/content-sidebar/activity-feed/activity-feed/ActiveState.js

```diff
@@ -187,7 +199,21 @@ const ActiveState = ({
                                 isFocused={isFocused}
                                 ref={refValue}
                             >
-                                <ActivityThread hasReplies={hasReplies} data-testid="activity-thread">
+                                <ActivityThread
+                                    data-testid="activity-thread"
```

Comment:
Is this needed since it's also in ActivityThread?

Reply:
It is - shallow render would not see the `data-testid` inside of the component

Reply:
Are there unit tests for `ActiveState` that rely on this `data-testid`?